### ISSUE TITLE
Relationships don't shop up when using Layout Editor

### ISSUE THREAD

**Issue by ghost** [CiviCooP/org.civicoop.relationship2summary#7]
There are no relationships showing up in overview. CiviCRM 5.67.2 with Drupal 102.2 (PHP 8.2) and Contact Editor installed.
Maybe a solution: https://github.com/CiviCooP/org.civicoop.relationship2summary/issues/4

**Comment by Upperholme** [comment]
Yes, it's a shame that the location of this relationship block is not controllable using the layout editor extension. That would be neat.
